third_party/move/move-model/src/builder/macros.rs

```diff
@@ -23,15 +27,30 @@ impl<'env> ModelBuilder<'env> {
     }
 
     fn expand_assert(&self, loc: Loc, args: &Spanned<Vec<EA::Exp>>) -> EA::Exp_ {
-        if args.value.len() != 2 {
-            self.error(
-                &self.to_loc(&args.loc),
-                "assert macro must have two arguments",
-            );
-            return EA::Exp_::UnresolvedError;
-        }
-        let cond = args.value[0].clone();
-        let abort_code = args.value[1].clone();
+        let (cond, abort_code) = match args.value.len() {
+            1 => {
+                self.check_language_version(
```

Comment:
Can we add a test in `third_party/move/move-compiler/tests/move_check/parser` to see this is guarded for V1? Does the V1 compiler use the move model for compilation?

Reply:
We already have a test in third_party/move/move-compiler/tests/move_check/naming/assert_one_arg.move.  I've added a separate warning in the V1 code to clarify that this is not supported in that compiler.